spec/integration/cart_spec.rb

```diff
@@ -201,16 +201,16 @@
             let(:line_item_resource) { build(:line_item_from_fixture) }
             it "should create a line item when requested using the line_items collection on the cart - using save" do
               # @TODO We should not have to specify the cart id
-              line_item = subject.line_items.new(item_id: variant.id, item_type: "Variant")
+              line_item = subject.line_items.new(item_id: variant.id, item_type: "Variant", cart_id: 1)
```

Comment:
Not a fan of having to specify the cart_id when doing `cart.line_items.new`, ideally it should get it from the original cart. If it's not possible then this is fine

Reply:
Should just be able to get the ID from the `subject` unless I'm missing something

Reply:
Also if we can make this change then the old TODO comment can be removed